test/user.js

```diff
@@ -0,0 +1,103 @@
+//During the test the env variable is set to test
+process.env.NODE_ENV = 'development';
```

Comment:
Why are you overriding this? Its intended that the node env is test while running the tests.